src/database/seeders/20240415080213-demo-user.ts

```diff
@@ -11,12 +11,7 @@ module.exports = {
           firstName: 'christian',
           lastName: 'Ishimwe',
           email: 'christianinja3@gmail.com',
-        },
-        {
-          userId: uuid(),
-          firstName: 'celestin',
-          lastName: 'Nshuti',
-          email: 'nshuticelestin@gmail.com',
+          password: 'christianinja3@gmail.com',
```

Comment:
Password should be hashed before getting into the database

Reply:
Since the user is being registered by google, and password field must be filled, i used google as default placeholder for that field, when  a user tries to sign in by email and password, will not not